airflow/www/extensions/init_views.py

```diff
@@ -58,6 +58,7 @@ def init_appbuilder_views(app):
     appbuilder.add_view(views.PoolModelView, "Pools", category="Admin")
     appbuilder.add_view(views.VariableModelView, "Variables", category="Admin")
     appbuilder.add_view(views.XComModelView, "XComs", category="Admin")
+    appbuilder.add_view(views.StatsView, "Stats", category="Admin")
```

Comment:
@ashb I added information about task instances and dag runs. When users use something else than celery executor then they will only see this simple info.

Reply:
It would be great if we could also get information from Kubernetes. We can do it separately, but the method's interface should be generic enough to be useful for KubernetesExecutor as well.

Reply:
Yeah, that's exactly what I was thinking.